webservices/schemas.py

```diff
@@ -699,6 +699,7 @@ def remove_fec_url(self, obj):
     fields={
         'beginning_image_number': ma.fields.Str(),
         'ending_image_number': ma.fields.Str(),
+        'html_url': ma.fields.Str(),
```

Comment:
Okay excellent, thanks!  So we shouldn't worry about the other schema at all then?